packages/sanity/src/core/releases/tool/detail/documentTable/useReleaseHistory.ts

```diff
@@ -51,7 +51,6 @@ export function useReleaseHistory(bundleDocuments: SanityDocument[]): {
   useEffect(() => {
     fetchAndParseAll()
     // When revision changes, update the history.
-    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Not related, but this is not necessary and it's returning error in the react-compiler check